test/e2e/tests/resize_e2e_test.go

```diff
@@ -41,7 +41,7 @@ const (
 	defaultTier          = "STANDARD"
 	defaultNetwork       = "default"
 	minVolumeSize  int64 = 1 * util.Tb
-	defaultEpsilon int64 = 70 * util.Gb // when a 1T instance was created, there will be 68-70 Gb of space unavailable
+	defaultEpsilon int64 = 70 * util.Gb // when a 1T instance was created, the actual size is 1007G, and only 956G is available to non-root users, so there will be ~70 Gb of space unavailable
```

Comment:
Should we instead count the actual size instead of space available to non-root users? Can a pod be root user as well? @saikat-royc

Reply:
as discussed let's report all